### DISCUSSION TITLE
M5StickC Plus Insulators  Wifi web build

### DISCUSSION THREAD

**Discussion by mikejohnau** [PlummersSoftwareLLC/NightDriverStrip#437] [General]
I dont seem to be able to create a wifi enabled insulators build. Also if I clone the git repository the build is way off.
I can generate a non wifi web build of the insulators env on the website. But I cant add wifi to the web build, the option never comes up in Chrome on MacOS. Even after following Dave's latest instructions.

I'm looking to edit out all of the effects except one. I have a very specific application. So eitherthe ability to build locally or the build the version with web serer enabled is the key.

**Comment by rbergen** [comment]
@mikejohnau I'm struggling a little with grasping some of the things you're mentioning (that's not an accusation, I'm probably just a bit slow on a Friday).  
Taking them one at a time:

- What do you mean the "build is way off" when you clone the repo? How does that manifest itself?
- You say you can generate a non wifi web build on the website. Do you mean you can flash a device with the insulators environment using the Web Installer (that being the one at <https://plummerssoftwarellc.github.io/NightDriverStrip/>)?
- Could you specify a little more precisely what "latest instructions" you followed? Is it something in the README.md? (If so, I may be able to help pinpoint and solve the confusion quickly, because I actually wrote most of the documentation that's now in the project.)
